src/widgets/chart.rs

```diff
@@ -353,7 +366,7 @@ impl<'a> Chart<'a> {
         layout
     }
 
-    fn max_width_of_labels_left_of_y_axis(&self, area: Rect) -> u16 {
+    fn max_width_of_labels_left_of_y_axis(&self, area: Rect, has_y_axis: bool) -> u16 {
```

Comment:
is there an advantage of having `has_y_axis` has a parameter instead of the doing `self.y_axis_labels.is_some()` within the method ?

Reply:
The fact that the Y axis is drawn only if it has labels seemed like a piece of logic that would not be obvious in this method.
It would also duplicate the code already present in `Chart::layout()`.

To avoid passing this paramter, another solution that I might propose is to encapsulate this logic in a new method `Chart::should_y_axis_be_rendered(&self)`, referred by both this method and `Chart::layout()`.

Reply:
That makes sense, thanks for the explanation. The new method might indeed be a nice addition but this is not blocking this PR.